WalletWasabi.Gui/Tabs/SettingsView.xaml

```diff
@@ -67,7 +67,7 @@
               <StackPanel Orientation="Vertical" Spacing="5">
                 <StackPanel Margin="0 10" Orientation="Horizontal" Spacing="5">
                   <ToggleButton IsChecked="{Binding Autocopy}" Content="{Binding Autocopy, Converter={StaticResource BooleanStringConverter}, ConverterParameter=On:Off}" Margin="0 0 10 0" />
-                  <TextBlock VerticalAlignment="Center">Autocopy on Receive and History wallet tabs.</TextBlock>
+                  <TextBlock VerticalAlignment="Center">Autocopy and pasting of bitcoin addresses, urls, and transaction ids.</TextBlock>
```

Comment:
Where do we paste transaction ids? Transaction data?

Automatically copy and paste of Textboxes when appropriate information found on clipboard?

Or something like this?

Reply:
We autocopy tx id - but yes, I don't think we paste it anywhere...